flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/utils/StreamTestData.scala

```diff
@@ -80,4 +83,28 @@ object StreamTestData {
     data.+=((5, 15L, 14, "KLM", 2L))
     env.fromCollection(data)
   }
+
+  def getPojo0DataStream(env: StreamExecutionEnvironment): DataStream[Pojo0] = {
+    val data =  new mutable.MutableList[Pojo0]
+    val p0 = new Pojo0
```

Comment:
We can make this more concise if we add two constructors to `Pojo0` (default without args and with all member variable).